src/components/google-maps-aerial.js

```diff
@@ -0,0 +1,144 @@
+import { TilesRenderer } from '3d-tiles-renderer';
+import {
+  TilesFadePlugin,
+  TileCompressionPlugin,
+  GLTFExtensionsPlugin,
+  GoogleCloudAuthPlugin
+} from '3d-tiles-renderer/plugins';
+
+console.log('3d-tiles-renderer', TilesRenderer);
+const MathUtils = AFRAME.THREE.MathUtils;
+const DRACOLoader = AFRAME.THREE.DRACOLoader;
+
+if (typeof AFRAME === 'undefined') {
+  throw new Error(
+    'Component attempted to register before AFRAME was available.'
+  );
+}
+
+AFRAME.registerComponent('google-maps-aerial', {
+  schema: {
+    apiToken: { type: 'string', default: '' },
+    latitude: { type: 'number', default: 37.795 }, // SF Ferry Building
+    longitude: { type: 'number', default: -122.394 },
+    minDistance: { type: 'number', default: 500 },
+    maxDistance: { type: 'number', default: 20000 },
+    ellipsoidalHeight: { type: 'number', default: 0 },
+    copyrightEl: { type: 'selector' }
+  },
+
+  init: function () {
+    this.tick = AFRAME.utils.throttleTick(this.tick, 10, this);
+    this.initialized = false;
+    // Initialize tiles
+    this.tiles = new TilesRenderer(
+      'https://tile.googleapis.com/v1/3dtiles/root.json'
+    );
+
+    // Register plugins
+    this.tiles.registerPlugin(
+      new GoogleCloudAuthPlugin({ apiToken: this.data.apiToken })
+    );
+    this.tiles.registerPlugin(new TileCompressionPlugin());
+    this.tiles.registerPlugin(new TilesFadePlugin());
+    this.tiles.registerPlugin(
+      new GLTFExtensionsPlugin({
+        dracoLoader: new DRACOLoader().setDecoderPath(
+          'https://unpkg.com/three@0.153.0/examples/jsm/libs/draco/gltf/'
+        )
+      })
```

Comment:
You should use
`this.el.sceneEl.systems["gltf-model"].getDRACOLoader();`
so you share the draco loader between the gltf-model component and this TilesRenderer.
That will use latest draco decoder 1.5.7 by default on aframe 1.7

Reply:
c9c1bdf95de6fd6e7f75a5a9f9fe71dbabffe97d